src/components/layout.js

```diff
@@ -28,19 +29,33 @@ const Layout = ({ children, active, bgColor }) => {
     }
   `)
 
+   // Used to position news page news feed on larger screens
+   const contentContainerRef = useRef()
+
   return (
     <>
-      <Header active={active} bgColor={bgColor} siteTitle={data.site.siteMetadata.title} />
+      <Header
+        active={active}
+        bgColor={bgColor}
+        siteTitle={data.site.siteMetadata.title}
+      />
       <main className={active}>
         <section className="main-wrapper">
           <Panel active={active} bgColor={bgColor} />
-          <div className="content-container">{children}</div>
+          <div className="content-container" ref={contentContainerRef}>
+            {children}
+          </div>
         </section>
       </main>
       {active === "about" && <Quotes />}
+      {active === "news" && (
+        <NewsFeed contentContainerRef={contentContainerRef} />
+      )}
       {active === "programs" && <WhereItStarted />}
       <FooterContactForm
```

Comment:
I don't know if my latest for the homepage is merged into your work here, but I also had to make some updates to the FooterContactForm component to mmake it usable on the homepage. Let's get together and make sure we are on the same page with this.